pages/en/glossary.mdx

```diff
@@ -0,0 +1,81 @@
+
+  1. **Active**: An allocation is considered active when it is created on-chain. The Indexer agent manages creating allocations based on the Indexer rules.
+
+  2. **Closed**: An Indexer can close an allocation once an epoch has passed, or their Indexer agent will automatically close the allocation after ~26 days. When an allocation is closed, their indexing rewards are distributed to the Indexer and its Delegators.
+
+  3. **Finalized**: Once an allocation has been closed, there is a dispute period, allowing an opportunity for a fisherman to open a dispute.
+
+  4. **Claimed**: The final phase of an allocation, all eligible rewards have been distributed and query fee rebates have been claimed.
+
+- **Subgraph Studio**: A powerful dapp for building, deploying, and publishing subgraphs.
+
+- **Fishermen**: Network participants that monitor query results served by Indexers. Fishermen are rewarded with GRT and receive 50% of the Indexer's indexing rewards for that epoch if they find an Indexer serving false data.
+
+- **Arbitrators**: Arbitrators guarantee network security by resolving disputes between Indexers and Fishermen.
+
+- **Slashing**: Indexers can have their rewards slashed for serving false data. The Indexer's self stake is slashed by 2.5% and the entirety of the rewards for that epoch are slashed as well. Additionally, 50% of the Indexer’s indexing rewards for that epoch are burned, and the other 50% are rewarded to the Fisherman.
+
+- **Indexing Rewards**: The rewards that Indexers receive for processing subgraphs. Indexing rewards are paid out in GRT.
+
+- **Delegation Rewards**: The rewards that Delegators receive for delegating GRT to Indexers. Delegation rewards are paid out in GRT.
+
+- **GRT**: The Graph's work utility token. GRT is used for providing economic incentives to network participants for contributing to the network.
+
+- **Indexing Node**: A server that runs an Indexer's indexer-agent software. Indexing nodes process subgraphs and make them available to be queried by subgraph consumers.
+
+- **Indexing Node Operators**: The people who run indexing nodes.
+
+- **POI or Proof of Indexing**: POIs are used to verify that an Indexer is indexing the subgraphs they have allocated on.
+
+- **Indexer-Agent**: The software that runs on an Indexing Node. It processes subgraphs and makes them available to be queried by subgraph consumers.
+
+- **Graph Explorer**: A dapp designed for network participants to explore subgraphs and interact with the protocol.
+
+- **Graph CLI**: A command line interface tool for building and deploying to The Graph.
```

Comment:
Maybe we should add [Graph Client](https://thegraph.com/docs/en/querying/querying-from-an-application/#graph-client) as well?

Reply:
Good call! Added that